packages/server/graphql/public/permissions.ts

```diff
@@ -67,7 +67,7 @@ const permissionMap: PermissionMap<Resolvers> = {
   },
   Query: {
     '*': isAuthenticated,
-    getDemoEntities: rateLimit({perMinute: 5, perHour: 50}),
+    getDemoGroupTitle: allow,
```

Comment:
-2 this endpoint could be hit by anyone as much as they want, they don't even need to be logged in. could this be abused by an attacker to cost us a ton of $$? are there any cheap  mitigation strategies we can implement?

Reply:
Added the rate limit!